components/tooltip/tooltip-help.js

```diff
@@ -44,6 +48,7 @@ class TooltipHelp extends FocusMixin(FocusVisiblePolyfillMixin(LitElement)) {
 			#d2l-tooltip-help-text {
 				background: none;
 				border: none;
+				cursor: text;
```

Comment:
This is a regression fix. Originally we had this automatically with a `span` opener, but it changed to the regular cursor after we started using a `button` for the opener. 

Let me know if I should revert the commit and make a separate PR for it.